lib/gems/pending/appliance_console/cli.rb

```diff
@@ -229,6 +240,34 @@ def set_external_db
       config.post_activation
     end
 
+    def set_replication_db
+      db_opts = {
+        :node_number       => options[:replication_node],
+        :database_name     => options[:dbname],
+        :database_user     => options[:username],
+        :database_password => options[:password]
+      }
+      case options[:replication]
+      when "primary"
+        db_replication = ApplianceConsole::DatabaseReplicationPrimary.new(db_opts)
```

Comment:
The initialize method for these classes does not take any options. This will fail in production.